examples/naming_package_files/readme.md

```diff
@@ -64,3 +64,41 @@ names_from_toolchains = rule(
 bazel build :example2
 ls -l bazel-bin/example2*.tar
 ```
+
```

Comment:
Consider renaming this file to `README.md` in a future change to match existing conventions.

Reply:
ack